src/app/component/Header/Header.container.js

```diff
@@ -229,20 +251,21 @@ export class HeaderContainer extends NavigationAbstractContainer {
     }
 
     onSearchOutsideClick() {
-        const {
-            goToPreviousNavigationState,
-            hideActiveOverlay,
-            navigationState: { name }
-        } = this.props;
+        const { goToPreviousNavigationState, navigationState: { name } } = this.props;
 
         if (!isMobile.any() && name === SEARCH) {
-            this.setState({ searchCriteria: '' });
-
-            hideActiveOverlay();
+            this.hideSearchOverlay();
             goToPreviousNavigationState();
         }
     }
 
+    goToDefaultHeaderState() {
+        const { setNavigationState } = this.props;
+        const state = this.getNavigationState(location.pathname);
```

Comment:
Why not to use the `location` from props?

Reply:
now uses `navigationState` from props